generators/app/prompts.js

```diff
@@ -9,6 +13,21 @@ module.exports = {
           name: 'seedbank',
           checked: true,
           value: "gem 'seedbank'"
+        },
+        {
+          name: 'rack-cors',
+          checked: true,
+          value: "gem 'rack-cors'"
+        },
+        {
+          name: 'redis',
+          checked: true,
+          value: "gem 'redis', '~> 3.0'"
+        },
+        {
+          name: 'bcrypt',
+          checked: true,
+          value: "gem 'bcrypt', '~> 3.1.7'"
```

Comment:
I actually was thinking of scrapping this altogether. I mean, this whole gem prompt. And instead, give the user the default Gemfile and then change to edit it. It would be simpler for us and more flexible for the users.

It was on my personal backlog to do when I had the time. What do you think?